src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -10,7 +10,7 @@
 import seedu.contax.model.person.Person;
 
 /**
- * Represents an appointment in the Schedule. Time related functionality is implemented in the superclass.
+ * Represents an appointment in the Schedule. Time related functionality is implemented in the superclass
```

Comment:
```suggestion
 * Represents an appointment in the Schedule. Time related functionality is implemented in the superclass.
```

Reply:
The sentence continues to the next line for the linking, which is why there is no fullstop here.